generate-stackbrew-library.sh

```diff
@@ -69,8 +69,10 @@ getArches 'solr'
 cat <<-EOH
 # this file is generated via https://github.com/docker-solr/docker-solr/blob/$(fileCommit "$self")/$self
 
-Maintainers: Martijn Koster <mak-github@greenhills.co.uk> (@makuk66),
-             Shalin Mangar <shalin@apache.org> (@shalinmangar)
+Maintainers: The Apache Lucene/Solr Project <solr-user@lucene.apache.org>,
+             Shalin Mangar (@shalinmangar),
+             David Smiley (@dsmiley),
+             Jan Høydahl (@janhoy)
```

Comment:
Oh come on; put yourself higher up :-)

Reply:
Strictly chronological :-)

This is just a temp change. Once we move the repo to ASF github, the maintiners list should ideally just be solr-user list? But the project does not have a github account for itself, so not sure if they will accept that part being empty?